src/SeriesSignup/SeriesSignup.vue

```diff
@@ -150,8 +150,12 @@ a {
   color: rgba(0, 0, 0, 0.667);
   opacity: 0.7;
 }
+.form__input__input:hover {
+  background-color: rgba(255,255,255,0.13);
+}
 .form__input__input:focus {
   outline: none;
+  box-shadow: inset 0 0 0 1.5px #66A1FF;
```

Comment:
Why are we using box-shadow here instead of border-color?

Also, I've increased the size of form field borders to 2px in the DS - apologies for not sharing the update.